x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/processor_settings_form/map_processor_type_to_form.tsx

```diff
@@ -5,52 +5,64 @@
  */
 
 import { FunctionComponent } from 'react';
+import { SetProcessor } from './processors/set';
+import { Gsub } from './processors/gsub';
 
-// import { SetProcessor } from './processors/set';
-// import { Gsub } from './processors/gsub';
-
-const mapProcessorTypeToForm = {
-  append: undefined, // TODO: Implement
-  bytes: undefined, // TODO: Implement
-  circle: undefined, // TODO: Implement
-  convert: undefined, // TODO: Implement
-  csv: undefined, // TODO: Implement
-  date: undefined, // TODO: Implement
-  date_index_name: undefined, // TODO: Implement
-  dissect: undefined, // TODO: Implement
-  dot_expander: undefined, // TODO: Implement
-  drop: undefined, // TODO: Implement
-  enrich: undefined, // TODO: Implement
-  fail: undefined, // TODO: Implement
-  foreach: undefined, // TODO: Implement
-  geoip: undefined, // TODO: Implement
-  grok: undefined, // TODO: Implement
-  html_strip: undefined, // TODO: Implement
-  inference: undefined, // TODO: Implement
-  join: undefined, // TODO: Implement
-  json: undefined, // TODO: Implement
-  kv: undefined, // TODO: Implement
-  lowercase: undefined, // TODO: Implement
-  pipeline: undefined, // TODO: Implement
-  remove: undefined, // TODO: Implement
-  rename: undefined, // TODO: Implement
-  script: undefined, // TODO: Implement
-  set_security_user: undefined, // TODO: Implement
-  split: undefined, // TODO: Implement
-  sort: undefined, // TODO: Implement
-  trim: undefined, // TODO: Implement
-  uppercase: undefined, // TODO: Implement
-  urldecode: undefined, // TODO: Implement
-  user_agent: undefined, // TODO: Implement
-
-  gsub: undefined,
-  set: undefined,
+/**
+ * Map that accepts an ES processor type name and returns either the component
+ * for rendering the associated fields for configuring the processor or the path
```

Comment:
> Map that accepts an ES processor type name and returns either the component
> for rendering the associated fields for configuring the processor or the path

This behavior feels a little weird to me. I think it would still be beneficial to link to the documentation even if the form is rendered.

Reply:
Yeah this was actually intended as a way to just tide us over until we have implemented the individual forms. But given that we want to display the link outside of the form perhaps it makes sense to convert these into objects with `link` and `component` entries.